client/web/src/enterprise/batches/detail/changesets/ExternalChangesetTitle.tsx

```diff
@@ -0,0 +1,24 @@
+import ExternalLinkIcon from 'mdi-react/ExternalLinkIcon'
+import React from 'react'
+
+import { LinkOrSpan } from '@sourcegraph/shared/src/components/LinkOrSpan'
+
+import { ExternalChangesetFields } from '../../../../graphql-operations'
+
+export const ExternalChangesetTitle: React.FunctionComponent<
+    Pick<ExternalChangesetFields, 'externalID' | 'externalURL' | 'title'> & {
+        /** Optionally, any class names to forward as a prop to the inner `LinkOrSpan` */
+        className?: string
+    }
+> = ({ className, externalID, externalURL, title }) => (
+    <LinkOrSpan to={externalURL?.url} target="_blank" rel="noopener noreferrer" className={`mr-2 ${className || ''}`}>
```

Comment:
We pull in [classnames](https://www.npmjs.com/package/classnames), which could theoretically simplify this:

```suggestion
    <LinkOrSpan to={externalURL?.url} target="_blank" rel="noopener noreferrer" className={classNames('mr-2', className)}>
```

(with an `import classNames from 'classnames'`, of course)

Reply:
Ooh sure! Thanks for the tip.